corehq/apps/case_search/utils.py

```diff
@@ -103,3 +104,20 @@ def _add_case_property_queries(self):
                 to_remove = re.escape(removal_regex.regex)
                 value = re.sub(to_remove, '', value)
             self.search_es = self.search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
+
+    def _add_daterange_queries(self):
+        # Add query for specially formatted daterange param
+        #   The format is __range__YYYY-MM-DD__YYYY-MM-DD, which is
+        #   used by App manager case-search feature
+        pattern = re.compile(r'__range__\d{4}-\d{2}-\d{2}__\d{4}-\d{2}-\d{2}')
```

Comment:
Does the daterange widget exist on Android? I'm assuming it also uses this formatted daterange parameter?

Reply:
Good question! @shubham1g5 would know.

Reply:
There is no such widget on Android right now.

Reply:
Okay, doesn't this break mobile?

Reply:
Mobile just treats any `input` types it don't recognize as free input text. So mobile should not crash but I think searches will break unless user enters this particular input format in the free text.

Reply:
I think that's unexpected. Let us flag and take this on the spec.

Reply:
Shubham has now implemented a way to ignore this field on mobile I think